onnxscript/function_libs/torch_aten/tensor_typing.py

```diff
@@ -54,6 +60,7 @@
 ]
 
 TTensor = TypeVar("TTensor", bound=_TensorType)
+TTensorOrString = TypeVar("TTensorOrString", bound=Union[_TensorType, str])
```

Comment:
Don't understand this. Do you mean a tensor of strings? If so, it should be `STRING` not `str`.

Reply:
Done